improver/cube_combiner.py

```diff
@@ -128,6 +128,34 @@ def process(self, *cubes: Union[Cube, CubeList]) -> Cube:
         return self.plugin(CubeList(filtered_cubes), self.new_name)
 
 
+def masked_add(
+    masked_array: np.ma.MaskedArray, masked_array_2: np.ma.MaskedArray
+) -> np.ma.MaskedArray:
+    """
+    Operation to add two masked arrays treating masked points as 0.
+
+    Args:
+        masked_array (numpy.ma.MaskedArray):
+            An array that may be masked.
+        masked_array_2 (numpy.ma.MaskedArray):
+            An array that may be masked.
+
+    Returns:
+        numpy.ma.MaskedArray:
+            The sum of the two masked arrays with masked points treated as 0.
+    """
+    new_array_1 = np.where(np.ma.getmask(masked_array), 0, masked_array)
+    new_array_2 = np.where(np.ma.getmask(masked_array_2), 0, masked_array_2)
```

Comment:
There's a simpler way:
```suggestion
    new_array_1 = np.ma.filled(masked_array), 0)
    new_array_2 = np.ma.filled(masked_array_2), 0)
```

Reply:
Done